### DISCUSSION TITLE
Error when installing… Something about "HEAD is now at f6ab300fc"

### DISCUSSION THREAD

**Discussion by hopelesscase** [Homebrew/discussions#3146] [Getting started] [answered]
So I messed up and uninstalled Homebrew and was trying to reinstall today when I was faced with the following message:

`HEAD is now at f6ab300fc Merge pull request #13085 from Homebrew/dependabot/bundler/Library/Homebrew/json_schemer-0.2.20
error: Not a valid ref: refs/remotes/origin/master
fatal: ambiguous argument 'refs/remotes/origin/master': unknown revision or path not in the working tree.
Use '--' to separate paths from revisions, like this:
'git <command> [<revision>...] -- [<file>...]'`

No idea what I'm suppose to do now.

I was using the copy-paste commands from brew.sh.

Not sure if others are getting that or not so I ended up here.

**Comment by SMillerDev** [comment, marked as answer]
Look at #666 for the answers

**Reply by hopelesscase** [reply]
I overlooked that somehow… :/ Thank you for your help and the fast reply. That fixed it.

**Reply by rupesh751** [reply]
> Look at #666 for the answers 

I WAS ALSO FACING THIS problem 

HEAD is now at 1a8f9ac70 Merge pull request #14008 from Bo98/bundler-arm-linux
error: Not a valid ref: refs/remotes/origin/master
fatal: ambiguous argument 'refs/remotes/origin/master': unknown revision or path not in the working tree.
Use '--' to separate paths from revisions, like this:
'git <command> [<revision>...] -- [<file>...]'

**Reply by rupesh751** [reply]
thank you so much for help and quick reply..
My problem was resolved..
thank you so much

**Comment by vksssd** [comment]
you might have already tried to install try to check version of brew 
vinayak@dont-know ~ % brew --version
Homebrew 3.4.0
Homebrew/homebrew-core (git revision 89308df497f; last commit 2022-03-06)
Homebrew/homebrew-cask (git revision a00efcd04d; last commit 2022-03-05)

if don't get this git repo 
try to troubleshoot 

Troubleshooting:
  brew config
  brew doctor
  brew install --verbose --debug FORMULA|CASK

or update: 
 brew update

**Comment by rupesh751** [comment]
i was also facing the same problem

> 2HEAD is now at 1a8f9ac70 Merge pull request #14008 from Bo98/bundler-arm-linux
error: Not a valid ref: refs/remotes/origin/master
fatal: ambiguous argument 'refs/remotes/origin/master': unknown revision or path not in the working tree.
Use '--' to separate paths from revisions, like this:
'git <command> [<revision>...] -- [<file>...]'

**Reply by SamuelMauricioL** [reply]
same

**Comment by SamuelMauricioL** [comment]
### That's work for me:

**1. Uninstall Homebrew Remnants**
`/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/uninstall.sh)"`

**2. Install again**
`/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"`

### Instalation log
<img width="595" alt="Screenshot 2022-11-16 at 06 23 30" src="https://user-images.githubusercontent.com/67238016/202167863-331f7ed5-d25e-497a-8987-ca84defce907.png">
